src/components/utilities.js

```diff
@@ -112,6 +113,18 @@ export const getTagLabelsArr = (tagsModel) => {
   return Object.values(tagRecords).map((tag) => tag.attributes);
 };
 
+export const getCustomLabelsArr = (customLabelsModel) => {
+  const customLabelsRecords = get(customLabelsModel, ['resolver', 'state', 'customLabel', 'records'], {});
```

Comment:
Shouldn't the default value be `[]` instead of `{}`?

Reply:
No, `records`  returns an object anyway, in the next string of this function we get values of this object
` return Object.values(customLabelsRecords).map(customLabel => customLabel.attributes);`